### DISCUSSION TITLE
CurrentSummationDelivered in ZbBridge Pro

### DISCUSSION THREAD

**Discussion by AK1Chel** [arendst/Tasmota#17889] [Q&A]
Please help. I am using ZbBridge Pro Ethernet Zigbee Gateway (ZB-GW03-V1.2). Tied Mi Power Plug ZigBee. I can turn it on and off, get power in watts. But the accumulated energy is displayed in kW. How to get the accumulated energy in watts?
22:23:04.311 MQT: tele/T28z/SENSOR = {"ZbReceived":{"0xFB19":{"Device":"0xFB19","Name":" Roz2","0000/FF01":"6410010328179839EC31D443953908982940052106009A2010082122110727000000000000000009210002","Power":1,"ActivePower":424,"CurrentSummationDelivered":2,"Endpoint":1,"LinkQuality":74}}}

**Comment by s-hadinger** [comment]
0000/FF01 is non standard Aqara specific. What is this device? What does `ZbInfo <device>` returns?

**Reply by AK1Chel** [reply]
I use 2 types of sockets. They both have the same answer. 
 [Xiaomi Mi Power Plug ZigBee ZNCZ02LM Zigbee compatibility (blakadder.com)](https://zigbee.blakadder.com/Xiaomi_ZNCZ02LM.html)     [Aqara Wall Outlet QBCZ11LM Zigbee compatibility (blakadder.com)](https://zigbee.blakadder.com/Aqara_QBCZ11LM.html)

08:40:57.353 MQT: tele/T28z/SENSOR = {"ZbReceived":{"0xFB19":{"Device":"0xFB19","Name":" Roz2","0000/FF01":"64100103281798390080EF4395395B492F40052106009A2010082122110727000000000000000009210002","Power":1,"ActivePower":479,"CurrentSummationDelivered":2,"Endpoint":1,"LinkQuality":68}}}
08:46:15.169 MQT: tele/T28z/SENSOR = {"ZbReceived":{"0x084D":{"Device":"0x084D","Name":" Roz1","0000/FF01":"64100103282D9839D7A3303F9539CEFC9C3D052171009A201008215A1309210002","Power":1,"ActivePower":0,"CurrentSummationDelivered":0,"Endpoint":1,"LinkQuality":128}}}

08:51:53.340 CMD: zbinfo
08:51:53.365 MQT: tele/T28z/SENSOR = {"ZbInfo":{"0x084D":{"Device":"0x084D","Name":" Roz1","IEEEAddr":"0x00158D00019D1143","ModelId":"lumi.plug","Manufacturer":"LUMI","Endpoints":[1,2,3],"Config":["O01","P01"],"Power":1,"ActivePower":0,"Reachable":true,"LastSeen":10,"LastSeenEpoch":1675655503,"LinkQuality":131}}}
08:51:53.378 MQT: tele/T28z/SENSOR = {"ZbInfo":{"0xFB19":{"Device":"0xFB19","Name":" Roz2","IEEEAddr":"0x00158D000320CE51","ModelId":"lumi.ctrl_86plug.aq1","Manufacturer":"LUMI","Endpoints":[1,2,3],"Config":["O01","P01","P02"],"Power":1,"ActivePower":470,"ActivePower2":481,"Reachable":true,"LastSeen":32,"LastSeenEpoch":1675655481,"LinkQuality":68}}}
